entries/d/delaware.gov.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Delaware.gov": {
+    "domain": "delaware.gov",
+    "tfa": [
+      "call",
+      "sms"
```

Comment:
From this super low res screenshot, it looks like they also support Okta Verify as an authentication method.
![image](https://github.com/user-attachments/assets/047aa22e-4c04-4a43-ac03-b869ef2001f6)

```suggestion
      "sms",
      "custom-software"
    ],
    "custom-software": [
      "Okta Verify"  
```

Reply:
Wow, thanks for the correction, I thought Okta was outside the scope of our 2FA methods! ❤️  @Carlgo11